web/firebase/firebase.js

```diff
@@ -0,0 +1,31 @@
+const firebase = require('firebase');
+// Required for side-effects
+require('firebase/firestore');
+
+const prodConfig = {
+  // TODO: setup production firestore project
+};
+
+const devConfig = {
+  apiKey: 'AIzaSyCBkQHeikIsiYZ2yOHiqH_mGJKDWMDU500',
```

Comment:
Keys!!!! Committed to source control!!!!!!! Noted in other PR but this should be loaded via env and stored in notion instead. For travis, this can be set via the UI or `.travis.yml`

Reply:
You can leave this config here for now cause we'll be using it for prod instead when I create a separate Firestore project for nwhacks2019-dev (that we should definitely keep in env).

Reply:
Can we restrict the keys so that they only work when coming from the site? Seems a bit risky to leave production keys out in the open

Reply:
Not a huge risk to leave it out in the open since they have to be public for anyone to interact with our db. Restricting by domain is a good idea though, I can do that after the dev project is setup (so we can still use this through localhost for now).